host/ProtocolGateway.Host.Fabric.FrontEnd/logging/Logger.cs

```diff
@@ -0,0 +1,304 @@
+﻿namespace ProtocolGateway.Host.Fabric.FrontEnd.Logging
+{
+    #region Using Clauses
+    using System;
+    using System.Diagnostics.Tracing;
+    using ProtocolGateway.Host.Fabric.FabricShared.Logging;
+    #endregion
+
+    /// <summary>
+    /// An event source used initiating ETW events
+    /// </summary>
+    [EventSource(Name = "Microsoft-ProtocolGateway-Application")]
+    public sealed class Logger : LoggerBase
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Initializes a new instance of the type
+        /// </summary>
+        /// <param name="systemName">The name of the system the logger is recording data for</param>
+        public Logger(string systemName) : base(systemName)
+        {
+        }
+
+        #endregion
+        #region Event Decorated Writers
+        /// <summary>
+        /// The method describing the verbose event for the ETW Manifest
+        /// </summary>
+        /// <param name="traceId">A unique identifier used to correlated related log messages</param>
+        /// <param name="computerName">The name of the computer the event is being written from</param>
+        /// <param name="processId">The process id that the event originated from</param>
+        /// <param name="systemName">The solution or service name that is logging the message</param>
+        /// <param name="componentName">The component within the clients solution that is logging the message</param>
+        /// <param name="message">The message to log</param>
+        /// <param name="errorMessage">The string representation of the exception that ocurred</param>
+        /// <param name="memberName">The member that logged the message within the client code</param>
+        /// <param name="fileName">The file name that logged the message within the client code</param>
+        /// <param name="lineNumber">The line number in the file that logged the message within the client code</param>
+        [Event(1, Keywords = Keywords.StandardMessage, Level = EventLevel.Verbose, Message = "{5}")]
+        protected override void VerboseStandardMessage(Guid traceId, string computerName, int processId,
+            string systemName, string componentName, string message, string errorMessage, string memberName,
+            string fileName, int lineNumber)
+        {
+            if (this.IsEnabled(EventLevel.Verbose, Keywords.StandardMessage))
```

Comment:
No, I'm asking if it makes sense to have a common method for this line as it keeps repeating. I see now there's different event level though so not much value.

---
In reply to: [73153473](https://github.com/Azure/azure-iot-protocol-gateway/pull/58#discussion_r73153473) [](ancestors = 73153473)